_posts/2019-02-26-data-security-of-salesforce.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Data Security Of Salesforce"
+date:   2019-02-26 22:27:00 +0800
+categories: blog
+author: Harry Hou
+---
+
+[Articles](https://trailhead.salesforce.com/content/learn/modules/data_security)
+
+## Data Security of Salesforce
+
+As a CRM application, it is important for the user to give the right people access to the right data. This post will introduce how to manage access control in Salesforce.
+
+### Levels of data access
+
+In Salesforce, you can control the level of access of your users to the data in your whole org, down to specific objects, fields, or even inidividual records.
+
+#### Control Access to the Organization
+
+For data security at organization level, what you can do is that ensure that only employees who meet certain criteria are allowed to log in to Salesforce. You can do this by managing authorized users, setting password policies, and limiting when and where users can log in.
+
+#### Control Access to Objects
+
+In Salesforce an object is a collection of records, like leads or contacts. So how to control access to objects? Here we must talk about **profiles**.
+
+A profile is a collection of settings and permissions. Every user in Salesforce has his own's profile, You can set object permissions with profiles or permission sets to control whether a group of users can create, view, edit, or delete any records of that object.
+
+#### Control Access to Fields
+
+After controlling `object-level` access, defining `field-level` security for sensitive fields is also an important part of Salesforce data security.
+
+In some cases, you want users to have access to an object, but limit their access to individual fields in that object. You can apply field settings by modifying profiles or permission sets just like the access control of objects.
```

Comment:
but limit their access => but ```to``` limit their access (this is to keep the tense the same: you want users "to have" access, but "to limit" their access to...)